style.css

```diff
@@ -280,8 +280,8 @@ ul.skill > li {
   align-items: flex-start;
   padding-left: 16px;
   gap: 10px;
-  margin-top: 0;
-  margin-bottom: 10px;
+  margin-top: 12px;
+  margin-bottom: 12px;
   padding-inline-start: 0;
   list-style: none;
 }
```

Comment:
@MaganaAsiati I have added the margin-top as requested, please. Thanks for the feedback.

Reply:
![added-margin-top](https://user-images.githubusercontent.com/32777408/200091325-de125e72-8c00-466c-9c40-70fd74a4bda2.png)

This is new look after adding the margin-top @MaganaAsiati